src/main/java/racingcar/controller/InputController.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.view.Input;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputController {
+    private static final String SPLIT_DELIM = ",";
+
+    public static List<Car> inputcars() {
+        return savecars();
+    }
```

Comment:
```suggestion
    public static List<Car> inputCars() {
        return saveCars();
    }
```
lowerCamelCase로 작성해주세요! (메서드 네이밍 관련하여 드린 링크에 lowerCamelCase 관련하여 설명이 나와있으니 참고해주세요.)

Reply:
수정하였습니다 :)